test/integration/smoke/test_annotations.py

```diff
@@ -135,6 +136,12 @@ def test_01_add_annotation(self):
         self.addAnnotation("annotation1", self.host.id, "HOST")
         self.assertEqual(self.added_annotations[-1].annotation.annotation, "annotation1")
 
+    @attr(tags=["devcloud", "advanced", "advancedns", "smoke", "basic", "sg"], required_hardware="false")
+    def test_01_add_ms_annotation(self):
+        """Testing the addAnnotations API ability to add an annoatation per management server"""
+        self.addAnnotation("annotation1", self.mgmt_server.id, "MANAGEMENT_SERVER")
+        self.assertEqual(self.added_annotations[-1].annotation.annotation, "annotation1")
```

Comment:
the added annotation is the same, maybe we can be unique about the comment here, for example:

```suggestion
        self.addAnnotation("mgmt-server-annotation1", self.mgmt_server.id, "MANAGEMENT_SERVER")
        self.assertEqual(self.added_annotations[-1].annotation.annotation, "mgmt-server-annotation1")
```

Reply:
Done